src/options/options.html

```diff
@@ -281,6 +281,35 @@ <h1 data-i18n="__MSG_titleSearchBar__">Search bar</h1>
 						alt="Example showing a search in the address bar with the term emoji “dog”, for which several emojis are suggested."
 						data-i18n data-i18n-alt="__MSG_optionImageAltTextSearchBarDemo__">
 					</div>
+
+					<li class="indent">
+						<div class="line">
+							<label data-i18n="optionEmojiSearchMaximumResults" for="maximumResults">Maximum results:</label>
+							<input class="setting save-on-input" type="range" value="9" min="1" max="21" step="1" id="maximumResults" data-optiongroup="emojiSearch" name="maximumResults">
+							<span>
+								<span id="maximumResultsStatus">(infinite)</span>
+							</span>
+						</div>
+						<span data-i18n="optionEmojiSearchMaximumResultsDescr" class="line indent helper-text">Limit the amount of emoji search results in the list. Drag to the maximum or unlimited results.</span>
+					</li>
+
+					<li class="indent">
+						<div class="line">
+							<input class="setting save-on-change" type="checkbox" id="enableFillingResults" data-optiongroup="emojiSearch" name="enableFillingResults">
+							<label data-i18n="enableFillingResults" for="enableFillingResults">Fill too few emoji results up with frequently used emojis.</label>
+						</div>
+						<span data-i18n="enableFillingResultsDescr" class="line indent helper-text">If enabled, the search results are followed by frequently used emojis.</span>
+
+						<div class="message-container">
+							<div id="tipFilling" aria-label="info message" data-i18n data-i18n-aria-label="ariaMessageInfo" class="message-box info">
```

Comment:
This setting is not applied directly/instantly when I change it:

https://github.com/user-attachments/assets/30c65b0c-887b-48c1-9095-150eac7ec25e

Probably because it is not send via the communication:
<img width="958" height="515" alt="Image" src="https://github.com/user-attachments/assets/e65985de-f5d7-42ab-9748-84ac198ce02e" />

Reply:
> Probably because it is not send via the communication:

Checking the code again this thesis is clearly wrong. This is just the trigger, it should then reload all needed settings via `AddonSettings`.